src/api/app/models/user_session.rb

```diff
@@ -0,0 +1,29 @@
+class UserSession
+  class FailedAuthentication < StandardError; end
+  class DisabledAccount < StandardError; end
+
+  # Returns the user storaged in current Thread
```

Comment:
I think this is evident. :rofl: I am personally against adding comments if they are not strictly needed. If the code is clear, most of the times the comments are not needed. :wink:

Reply:
:sweat_smile: :sweat_smile: :sweat_smile: 

You're right.

I did that with the generation of docs in mind since it is something that I am doing for some time now.

That said, I am going to remove them :)